utils/cronjobs_osgeo_lxd/README.md

```diff
@@ -1,5 +1,18 @@
 # Cronjobs OSGeo LXD
 
+## Version overview
+
+| **label**   | **version number** |
+| ----------- | ------------------ |
+| legacy      | 7.8                |
+| old current | 8.2                |
+| new current | 8.3                |
+| preview     | 8.4                |
```

Comment:
Perhaps use the table to explain the terms and leave the specific numbers just in the scripts. That way, the table won't need any updates, but it will be actual useful for explaining the reasoning behind the numbers and names.

Reply:
Polished in 15f4a5db51b364e20b2af017e695b5434a2a2794